Tasks/Common/AzureRmDeploy-common/azure-arm-rest/azure-arm-app-service-kudu.ts

```diff
@@ -592,21 +601,17 @@ export class Kudu {
         }
     }
 
-    private async _getDeploymentDetailsFromPollURL(pollURL: string,  affinityCookie?: string[]):Promise<any> {
+    private async _getDeploymentDetailsFromPollURL(pollURL: string):Promise<any> {
         let httpRequest = new webClient.WebRequest();
         httpRequest.method = 'GET';
         httpRequest.uri = pollURL;
         httpRequest.headers = {};
 
-        if(!!affinityCookie) {
-           httpRequest.headers['set-cookie'] = affinityCookie; 
-        }
-
         while(true) {
             let response = await this._client.beginRequest(httpRequest);
             if(response.statusCode == 200 || response.statusCode == 202) {
                 var result = response.body;
-                tl.debug(`POLL URL RESULT: ${JSON.stringify(result)}`);
+                tl.debug(`POLL URL RESULT: ${JSON.stringify(response)}`);
```

Comment:
why this change?

Reply:
pritning the whole response to check if the call hits the same kudu instance.